SYCL/ESIMD/regression/dgetrf.cpp

```diff
@@ -6,7 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-// REQUIRES: gpu
+// REQUIRES: gpu, aspect-fp64
```

Comment:
Do we define `aspect-fp64` anywhere? I'm concerned that this change will simply disable those tests

Reply:
I've copied from a now obsolete PR, but maybe I didn't fully understand it and missed that fact. Thanks for noticing - I'll verify if it's defined later today.

Reply:
You're totally right - it doesn't exist. I'll change to make runtime `device.has` check.